hyperliquid/exchange.py

```diff
@@ -509,6 +633,18 @@ def spot_transfer(self, amount: float, destination: str, token: str) -> Any:
         )
 
     def withdraw_from_bridge(self, amount: float, destination: str) -> Any:
+        """
+        Initiate a withdrawal from the Hyperliquid bridge to a specified destination.
+
+        Args:
+            amount (float): The amount to withdraw. This should be in the base unit of the asset
```

Comment:
I found this confusing. All withdrawals are in USDC
```suggestion
            amount (float): The amount of USD to withdraw.
```